x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx

```diff
@@ -378,31 +378,18 @@ export function useOnSubmit({
 
       const hasGoogleCloudShell = data?.item ? getCloudShellUrlFromPackagePolicy(data.item) : false;
 
-      if (hasFleetAddAgentsPrivileges) {
```

Comment:
Wow!  This is great.  I'm really curious why we don't need this anymore.

Reply:
Thank you! That's a good callout. @opauloh  I saw in the git history you add the conditions for showing the template prompts. I removed the [pop-up condition logic ](https://github.com/elastic/kibana/blob/c8edac6e3f2630ef762275e1cfcf95fb198b40f5/x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx#L381) because I believe we are repeating the [same logic below](https://github.com/elastic/kibana/blob/c8edac6e3f2630ef762275e1cfcf95fb198b40f5/x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx#L404). `SUBMITTED` is used to set status which indicates package policy has been successfully saved or the agent count exists.

Reply:
I think your updated logic below to check for `hasFleetAddAgentsPrivileges` has covered this piece of logic @Omolola-Akinleye, nice catch